DEV-9/EntryPoint.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using static System.Console;
+namespace RandomChangeTheSymbolsInString
+{
+  //The entry point of the program
+  class EntryPoint
+  {
+    //This method is used for calling the method for substitution
+    //random amount of symbols from random index in the source line to another string 
+    static void Main()
+    {
+      const string SOURCELINE = "QWERTYUIOP";
+      const string LINEFORINSERTION = "asdfghjk";
+
+      string sourceLine = SOURCELINE;
```

Comment:
Why you do not use SOURCELINE  and LINEFORINSERTION  directly? Unnecessary local values